src/main/java/de/tum/in/www1/artemis/service/connectors/localci/LocalCIContainerService.java

```diff
@@ -67,6 +67,18 @@ public class LocalCIContainerService {
     @Value("${artemis.continuous-integration.local-cis-build-scripts-path}")
     String localCIBuildScriptBasePath;
 
+    @Value("${artemis.continuous-integration.proxies.use-system-proxy:false}")
+    boolean useSystemProxy;
+
+    @Value("${artemis.continuous-integration.proxies.default.http-proxy}")
+    String httpProxy;
```

Comment:
final String maybe?

Reply:
@Value annotated variables never seem to be final in our code at least. Otherwise you get some errors